.pipelines/templates/validate-labels.yaml

```diff
@@ -0,0 +1,25 @@
+steps: 
+- bash: |
+    test_label=$(kubectl get namespace kube-system -o jsonpath="{.metadata.labels.openservicemesh\.io\/ignore}")
```

Comment:
Is there a reason for the tests being part of the pipelines instead of a separate tests dir that's invoked by the pipelines?

Reply:
Keep these in test/bats and call the scripts from the pipeline